internal/nodes/avi_model_l7_translator.go

```diff
@@ -49,14 +50,9 @@ func (o *AviObjectGraph) BuildL7VSGraph(vsName string, namespace string, ingName
 	if err != nil {
 		o.DeletePoolForIngress(namespace, ingName, key, vsNode)
 	} else {
-		ingObj, ok := utils.ToNetworkingIngress(myIng)
-		if !ok {
-			utils.AviLog.Errorf("Unable to convert obj type interface to networking/v1beta1 ingress")
-		}
-
 		var parsedIng IngressConfig
 		processIng := true
-		processIng = filterIngressOnClass(ingObj) && utils.CheckIfNamespaceAccepted(namespace, utils.GetGlobalNSFilter(), nil, true)
+		processIng = validateIngressForClass(key, ingObj) && utils.CheckIfNamespaceAccepted(namespace, utils.GetGlobalNSFilter(), nil, true)
```

Comment:
@akshayhavile Please check if this works fine with the newly introduced Changes for migration.